.vsts-ci/mac.yml

```diff
@@ -53,7 +53,8 @@ phases:
 
   # Uploads any packages as an artifact
   - powershell: |
-      Get-ChildItem -Path *.pkg, *.tar.gz, *.zip -Recurse | Select-Object -ExpandProperty FullName | ForEach-Object {
+      Write-Host "PWD == $pwd"
```

Comment:
As far as I remember `$pwd` variable should Not be used in scripts. Change to Get-Location.

Reply:
it's likely in this context it's ok, but something that we should be aware of.